test/user-test.js

```diff
@@ -0,0 +1,147 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../server.js';
+
+// ASSERTION STYLE
+chai.should();
+
+chai.use(chaiHttp);
+
+describe('Admin can create an employee user account and both admin/employee can login', () => {
+  // create-user test
+  describe('POST /api/v1/auth/create-user', () => {
```

Comment:
why do we have two describe??

Reply:
i am describing two post request instances; signup and login